docs/apache-airflow-providers-trino/operators/trino.rst

```diff
@@ -17,19 +17,22 @@
 
 .. _howto/operator:TrinoOperator:
 
-TrinoOperator
-=============
+Connect to Trino using SQLExecuteQueryOperator
+==============================================
 
-Use the :class:`TrinoOperator <airflow.providers.trino.operators.trino>` to execute
+Use the :class:`SQLExecuteQueryOperator <airflow.providers.common.sql.operators.sql>` to execute
 SQL commands in a `Trino <https://trino.io/>`__ query engine.
 
+.. warning::
+    Previously, TrinoOperator was used to perform this kind of operation. But at the moment TrinoOperator is deprecated and will be removed in future versions of the provider. Please consider to switch to SQLExecuteQueryOperator as soon as possible.
+
```

Comment:
We can just say: TrinoOperator is deprecated in favor of SQLExecuteQueryOperator. If you are using TrinoOperator you should migrate as soon as possible.

Reply:
yes as suggested by @Taragolis for postgresoperator, I kept the text same across all the documentation. I can rephrase it this way for better understanding

Reply:
No strong opinion here. I will resolve this thread